packages/utils/src/require-hooks/require-hooks.ts

```diff
@@ -1,4 +1,13 @@
+import fs from 'fs';
+import path from 'path';
+
 export function registerRequireHooks() {
-  require('@ts-tools/node');
-  require('@stylable/node/register');
+  const projectPath = process.cwd();
```

Comment:
We are also using the projectPath in the CLI commands. Should we receive as a prop to avoid duplication?

Reply:
I think so. 
Regarding documnetation it - we definitly need to document it! we're adding a way to configure our tools so we must write it down in the README